xym/xym.py

```diff
@@ -143,12 +143,12 @@ def get_extracted_models(self, force_revision_pyang, force_revision_regexp):
             models.extend(self.extracted_models)
             for model in models:
                 if force_revision_pyang:
-                    command = '/usr/local/bin/pyang -f name-revision "' + self.dst_dir + '/' + model + '"'
+                    command = '/usr/local/bin/pyang -f name --name-print-revision "' + self.dst_dir + '/' + model + '"'
```

Comment:
Having '/usr/local/bin' hardcoded seems unwise. What if user is using a virtualenv, for example? That is what I do all the time, and so this would immediately break for me. Should check for pyang availability on path, then perhaps fallback to this path?